src/external/coremltools_wrap/coremltools/mlmodel/build/format/OneHotEncoder_enums.h

```diff
@@ -21,6 +21,7 @@ static const char * MLOneHotEncoderCategoryType_Name(MLOneHotEncoderCategoryType
         case MLOneHotEncoderCategoryType_NOT_SET:
             return "INVALID";
     }
+    return "INVALID";
```

Comment:
If the compiler is generating a false negative on the return values being present, probably easier to just conditionally turn off that warning if the compiler in bad.  This directory was meant to be rsync clobbered by the coremltools directory.

Reply:
Good point. Since I already have tests passing on this one, let's put it in as is (assuming we don't need further commits), and open a GitHub issue to track the additional warnings (which will be fixed by the time we need to do another rsync).

Reply:
This is already fixed in coremltools with https://github.com/apple/coremltools/pull/332.